backend/index.js

```diff
@@ -96,17 +97,28 @@ app.post("/login", async (req, res) => {
   }
 });
 
-// Protected route example
-app.get("/protected", authenticateToken, async (req, res) => {
-  try {
-    const user = await prisma.user.findUnique({ where: { id: req.user.id } });
-    if (!user) {
-      return res.status(401).json({ message: "User not found" });
+//Endpoint for fethcing businesses
+app.get("/api/businesses", (req, res) => {
+  const jsonFilePath = path.join(
```

Comment:
Nit: we don't need to do this `path.join()` computation on every request since its inputs are static (always the same), so we can do it at the top-level scope (e.g. line 99).  However, the performance impact is negligible so you don't need to worry about this.